pkg/provider/azure_loadbalancer.go

```diff
@@ -2198,7 +2206,7 @@ func (az *Cloud) buildHealthProbeRulesForPort(serviceManifest *v1.Service, port
 			}
 		}
 		if path == nil {
-			path = pointer.String(consts.HealthProbeDefaultRequestPath)
+			path = pointer.String("/healthz")
```

Comment:
What is the default request path? Is it not already `/healthz`?

Reply:
No the default is `/` Do we want to change that at the const level?

Reply:
Nowhere else.

Reply:
Then I would change it yes :)